### ISSUE TITLE
Health check endpoint not in API documentation

### ISSUE THREAD

**Issue by cambraca** [mastodon/documentation#1174]
I found out about the /health endpoint from this page: https://blog.joinmastodon.org/2019/10/mastodon-3.0-in-depth/#health-check

Expected: a mention of it in the API docs (https://docs.joinmastodon.org/api/).

**Comment by innosflew** [comment]
Yeah, I also only happened to find out about this by mere chance on that same blog post. This is really should be included in the documentation somewhere.

**Comment by urda** [comment]
This looks like the PR it was added: https://github.com/mastodon/mastodon/pull/11770
